velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -270,7 +270,11 @@ std::vector<RowVectorPtr> AggregationFuzzerBase::generateInputData(
     generator = findInputGenerator(signature.value());
   }
 
-  const auto size = vectorFuzzer_.getOptions().vectorSize;
+  auto size = vectorFuzzer_.getOptions().vectorSize;
+  // 40% of times test aggregation fuzzer on empty input vectors.
+  if (vectorFuzzer_.coinToss(0.4)) {
+    size = 0;
+  }
```

Comment:
Why do we need this change? `vectorFuzzer_.getOptions().vectorSize` comes from FLAGS_batch_size, so we don't need to randomize it again.

Reply:
The batch_size flag is not modified in the Github fuzzer workflow and the default value of 100 is used for all fuzzer tests. The size is set to 0 here so the aggregate fuzzer is tested on empty input so the issue https://github.com/facebookincubator/velox/issues/7029 is resolved, since the changes from https://github.com/facebookincubator/velox/issues/7029 are included in this PR. Could you please share if this is fine, or whether it should be set randomly for fuzzer tests in `.github/workflows/experimental.yml`?

Reply:
Hi @pramodsatya, I discussed with @kgpai. We think it may be better to make it a unit test for #7100. Please see the comment here: https://github.com/facebookincubator/velox/issues/7029#issuecomment-2111350383.